libr/anal/class.c

```diff
@@ -0,0 +1,1129 @@
+/* radare - LGPL - Copyright 2018 - thestr4ng3r */
+
+#include <r_anal.h>
+#include <r_vector.h>
+#include "../include/r_anal.h"
+
+#define CLASSES_FLAGSPACE "classes"
+
+static char *sanitize_id(const char *id) {
+	if (!id || !*id) {
```

Comment:
Don't we have a macro for this?

Reply:
Hmm, there is `r_str_sanitize()`, but it doesn't remove `.` chars, which is needed to prevent corrupting the sdb structure.